q2_diversity/_alpha.py

```diff
@@ -5,3 +5,100 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
+
+import os
+
+import biom
+import scipy
+import skbio
+import qiime
+import pandas as pd
+import seaborn as sns
+import skbio.diversity
+from statsmodels.sandbox.stats.multicomp import multipletests
+
+
+def _get_phylogenetic_metrics():
+    return ('faith_pd')
+
+
+def alpha_diversity_phylogenetic(metric: str, feature_table: biom.Table,
+                                 phylogeny: skbio.TreeNode=None) -> pd.Series:
+    if metric not in _get_phylogenetic_metrics():
+        raise ValueError('%s is not a known phylogenetic metric.' % metric)
+
+    counts = feature_table.matrix_data.toarray().astype(int).T
+    sample_ids = feature_table.ids(axis='sample')
+    feature_ids = feature_table.ids(axis='observation')
+
+    return skbio.diversity.alpha_diversity(
+              metric=metric,
+              counts=counts,
+              ids=sample_ids,
+              otu_ids=feature_ids,
+              tree=phylogeny)
+
+
+def alpha_diversity(metric: str, feature_table: biom.Table) -> pd.Series:
+    if metric in _get_phylogenetic_metrics():
+        raise ValueError('%s is a phylogenetic diversity metric. Only non-'
+                         'phylogenetic metrics can be used with this method.'
+                         % metric)
+
+    counts = feature_table.matrix_data.toarray().astype(int).T
+    sample_ids = feature_table.ids(axis='sample')
+
+    return skbio.diversity.alpha_diversity(
+              metric=metric,
+              counts=counts,
+              ids=sample_ids)
+
+
+def alpha_diversity_comparisons(output_dir: str, alpha_diversities: pd.Series,
```

Comment:
I'm open to better names for this - it generates boxplots and some associated statistics.